com.unity.ml-agents/Runtime/Sensor/RayPerceptionSensor.cs

```diff
@@ -308,8 +340,7 @@ public virtual SensorCompressionType GetCompressionType()
         /// Evaluates the raycasts to be used as part of an observation of an agent.
         /// </summary>
         /// <param name="input">Input defining the rays that will be cast.</param>
-        /// <param name="output">Output class that will be written to with raycast results.</param>
-        ///
+        /// <returns>Output class that will be include the raycast results.</returns>
```

Comment:
```suggestion
        /// <returns>Output struct containing the raycast results.</returns>
```
Open to suggestions here, but "will be include the raycast results" is bad :)

Reply:
Much better.